ota-plus-web/app/reactapp/style/ota-default/variables-ota.scss

```diff
@@ -52,6 +53,10 @@ $sw-repo-border-color: #cfd0d3;
 $confirm-modal-title: #d0021b;
 $olp-group-layer: $warning-color;
 
+//size constants
+$main-content-height: calc(100vh - (50px + 60px)); //100vh - (navigation bar + legal info footer)
```

Comment:
How about we create separate `scss` variables for main `header`and `footer` heights? In case these values change in the future, it'll be much easier to manipulate them.

Reply:
Done